src/include/cell.i

```diff
@@ -112,8 +115,15 @@ __cell_pack_value_validity(WT_SESSION_IMPL *session, uint8_t **pp, WT_TIME_WINDO
             LF_SET(WT_CELL_TS_DURABLE_STOP);
         }
     }
-    if (tw->prepare)
+    if (LF_ISSET(
```

Comment:
Is there a reason you assign a bool on the reconcile structure rather than just incrementing that statistic here?

Reply:
We want to log stats for a page if at least there is one cell with ts/txnid/prepared data. If we increment logs here, we are counting cells on a page rather than the pages.